crates/biome_project/tests/invalid/lisence_not_string.json.snap

```diff
@@ -4,7 +4,7 @@ expression: lisence_not_string.json
 ---
 lisence_not_string.json:2:13 deserialize ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
 
-  × The value of key license is incorrect. Expected a string.
+  × Incorrect type, expected a string.
```

Comment:
It's a bit sad that the message is generic :(

Reply:
Yeah. I am not sure how to fix it. I have to think about that.

Reply:
Yeah. I am not sure how to fix it without making the API more complex. I have to think about that.

Reply:
I think it's fine for now, I understand the drawbacks of the refactor